src/event_loop.rs

```diff
@@ -97,7 +97,10 @@ impl<T> EventLoopBuilder<T> {
     ///   `WINIT_UNIX_BACKEND`. Legal values are `x11` and `wayland`.
     ///   If it is not set, winit will try to connect to a Wayland connection, and if that fails,
     ///   will fall back on X11. If this variable is set with any other value, winit will panic.
+    /// - **Android:** Must be configured with an `AndroidApp` from `android_main()` by calling
+    ///     [`.with_android_app(app)`] before calling `.build()`.
     ///
+    /// [`.with_android_app(app)`]: crate::platform::android::EventLoopBuilderExtAndroid::with_android_app
```

Comment:
You can do something like:
```suggestion
    #[cfg_attr(target_os = "android", doc = "[`.with_android_app(app)`]: crate::platform::android::EventLoopBuilderExtAndroid::with_android_app")]
    #[cfg_attr(not(target_os = "android"), doc = "[`.with_android_app(app)`]: #only-available-on-android")]
```

Reply:
ah, neat, thanks!